code/modules/mob/living/simple_animal/simple_animal.dm

```diff
@@ -615,3 +615,8 @@
 	if(pcollar && collar_type)
 		add_overlay("[collar_type]collar")
 		add_overlay("[collar_type]tag")
+
+/mob/living/simple_animal/Login()
+	. = ..()
```

Comment:
Any reason why you did `. = ..()`? afaik `Login()` doesnt take a return value anywhere, so this seems redundant

Reply:
Changed